iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -528,7 +528,7 @@ public TSExecuteStatementResp executeStatement(TSExecuteStatementReq req) throws
         return executeUpdateStatement(physicalPlan);
       }
     } catch (Exception e) {
-      LOGGER.warn("meet error: {}  while executing statement: {}", e.getMessage(), req.getStatement());
+      LOGGER.info("meet error: {}  while executing statement: {}", e.getMessage(), req.getStatement());
```

Comment:
Why change this to info level

Reply:
Not changed by me, it should be change from master branch when I merge it.